PORT-2479 Plenty of docs fixes and updates (#151)
port-labs/port-docs: docs/complete-use-cases/sdlc.md

}
```

> After creating the initial service Blueprint, go back and edit the Blueprint again, but replace the `relations` key with the following `relations` object:

Reviewer: ```suggestion
After creating the initial service Blueprint, return to the Blueprint and replace the `relations` key with the following `relations` object:
```

  helmLink: "https://github.com/port-labs/helm/notification.yml"
```

:::info

> Recall that you created a Relation from the service Blueprint to itself, allowing you to map dependencies between your services. If you want to add a service that `notification-service` depends on, you can do that by adding the following `relations` key to the `port.yml` (some services were added as an example):

Reviewer: ```suggestion
Before, you created a Relation from the service Blueprint to itself. That Relation allowed you to map dependencies between your services. If you want to add a service on which `notification-service` depends, add the following `relations` key to the `port.yml` (notice some services were added as an example):
```
